packages/@react-native-windows/cli/src/commands/initWindows/initWindows.ts

```diff
@@ -29,7 +29,7 @@ import {
   endTelemetrySession,
 } from '../../utils/telemetryHelpers';
 import {copyAndReplaceWithChangedCallback} from '../../generator-common';
-import * as nameHelpers from '../../utils/nameHelpers';
+import * as nameHelpers from '@react-native-windows/telemetry/src/utils/nameHelpers';
```

Comment:
Look at telemetry/index.ts to see how to export the functions we need there (`isValidProjectNamespace`, `isValidProjectName`, `cleanName`, `cleanNamespace`). Then you can import those functions here with:

```suggestion
import {isValidProjectNamespace, isValidProjectName, cleanName, cleanNamespace} from '@react-native-windows/telemetry';
```